pkg/catalog/manager/traverse.go

```diff
@@ -25,6 +25,7 @@ func (m *Manager) traverseAndUpdate(helm *helmlib.Helm, commit string, cmt *Cata
 	catalogType := getCatalogType(cmt)
 
 	index, err := helm.LoadIndex()
+	logrus.Debugf("helm index hash: %v", index.Hash)
```

Comment:
if this is only for debugging purposes, you think adding catalog/chart name/info would help?